crates/matrix-sdk-crypto/CHANGELOG.md

```diff
@@ -32,16 +32,20 @@ Changes:
 
 Breaking changes:
 
+  **NOTE**: this version causes changes to the format of the serialised data in
+  the CryptoStore, meaning that, once upgraded, it will not be possible to roll
+  back applications to earlier versions without breaking user sessions.
+
+- Change the structure of the `SenderData` enum. The new `SenderDataReader`
+  demonstrates how to migrate from the previous form.
+  ([#3877](https://github.com/matrix-org/matrix-rust-sdk/pull/3877))
```

Comment:
We need to actually document the behaviour change, as well as the interface change. As I understand it, `OlmMachine.get_encryption_info` will now return a different `VerificationState` if a user who was previously verified changes their identity?

Reply:
Fixed in 146656105d